js/search.js

```diff
@@ -949,25 +949,38 @@
 			for (var i = 0; i < moves.length; i++) {
 				var id = moves[i];
 				var isViable = BattleMovedex[id] && BattleMovedex[id].isViable;
-				if (id === 'aerialace') isViable = (toId(set.ability) === 'technician');
+				if (id === 'aerialace') isViable = (toId(set.species) in {scyther:1, aerodactylmega:1, kricketune:1});
 				if (id === 'bellydrum') isViable = (toId(set.species) in {azumarill:1, linoone:1, slurpuff:1});
-				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1});
+				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1, alakazam:1});
+				if (id === 'curse') isViable = (toId(set.species) === 'snorlax');
+				if (id === 'drainingkiss') isViable = (toId(set.ability) === 'triage');
 				if (id === 'dynamicpunch') isViable = (toId(set.ability) === 'noguard');
+				if (id === 'electroball') isViable = (toId(set.species) === 'raichualola');
+				if (id === 'endeavor') isViable = (toId(set.species) in {mamoswine:1, scolipede:1});
 				if (id === 'focuspunch') isViable = (toId(set.species) === 'breloom');
 				if (id === 'gyroball') isViable = (template.baseStats.spe <= 60);
+				if (id === 'headbutt') isViable = (toId(set.species) === 'dunsparce');
```

Comment:
Why not Serene Grace?

Reply:
Is Togekiss good with headbutt, also does it even get it?

Reply:
I think dunsparce is the only one to viably run it, the others are either special, have better things to do (jirachi), or are at awkward speed tier to try (para/)flinch (sawsbuck)

Reply:
Usable Moves is expressly not intended to replace viable movelists. You can make the argument that Iron Head mostly outclasses Headbutt on Jirachi, and maybe that Air Slash outclasses it on Togekiss, but Deerling/Sawsbuck can do things with it. And Togepi might be able to do something interesting with a Hustle set.

Reply:
off 20 attack and unstabbed? sg as the param wouldn't help hustle toge anyway. is dunsparce+deers enough? i guess jirachi maaaaybe and that adding it would make this push it but i dont think the broad serenegrace->headbutt is not outclassed link is there

Reply:
Oh, good point. Maybe Serene Grace + STAB?

Reply:
great